### PR TITLE
updated the bidding api with changes as per docs

### PR DESCRIPTION
**Pull request by uckamdar59**
Check and verify

### CODE REVIEW

File: biddingApi/biddingApi/src/main/java/biddingApi/biddingApi/Service/BiddingServiceImpl.java [hunk 1]
```diff
@@ -20,226 +19,248 @@
...
+					response.setStatus(Constants.unitValueisNull);
+					return response;
+				} else {
+					data.setUnitValue(bidPutRequest.getUnitValue());
+					data.setRate(bidPutRequest.getRate());
+					if (bidPutRequest.getBiddingDate() != null) {
+						data.setBiddingDate(bidPutRequest.getBiddingDate());
+					}
+
+					if (bidPutRequest.getTruckId() != null) {
```
Comment (badireddisatyasaivarun) [inline biddingApi/biddingApi/src/main/java/biddingApi/biddingApi/Service/BiddingServiceImpl.java:217]: truckId cannot be updated by shipper

File: biddingApi/biddingApi/src/main/java/biddingApi/biddingApi/Service/BiddingServiceImpl.java [hunk 2]
```diff
@@ -20,226 +19,248 @@
...
+						data.setTruckId(bidPutRequest.getTruckId());
```
Comment (badireddisatyasaivarun) [inline biddingApi/biddingApi/src/main/java/biddingApi/biddingApi/Service/BiddingServiceImpl.java:218]: .

**Review by badireddisatyasaivarun — COMMENTED** [review]
change these comments

File: biddingApi/biddingApi/src/main/java/biddingApi/biddingApi/Service/BiddingServiceImpl.java [hunk 3]
```diff
@@ -20,226 +19,248 @@
...
-		}
-		
-		if(bidPutRequest.getUnitValue()!=null)
-		{
-			if("PER_TON".equals(String.valueOf(bidPutRequest.getUnitValue())))                /////////////////////////////////////////////
-			{
-				//PER_TON, PER_TRUCK
-				dataById.setUnitValue(UnitValue.PER_TON);
+
+		if (String.valueOf(bidPutRequest.getTransporterApproval()).equals("true")
```
Comment (badireddisatyasaivarun) [inline biddingApi/biddingApi/src/main/java/biddingApi/biddingApi/Service/BiddingServiceImpl.java:143]: 1. what if the transporterApproval and shipperApproval both are NULL
2. what if the transporterApproval and shipperApproval both are not NULL

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**